src/serac/physics/utilities/functional/domain_integral.hpp

```diff
@@ -166,42 +92,36 @@ void evaluation_kernel(const mfem::Vector& U, mfem::Vector& R, derivatives_type*
 
     // for each quadrature point in the element
     for (int q = 0; q < static_cast<int>(rule.size()); q++) {
-      // get the position of this quadrature point in the parent and physical space,
```

Comment:
You removed a comment here, is it no longer needed? valid? was this a mistake?

Reply:
I'll put it into the combined host_device function.